browser/js/components/Comment/test.js

```diff
@@ -0,0 +1,151 @@
+'use strict'
+
+import React, {
```

Comment:
@thejohnbackes @jdhang @jancodes @jes708 
Omri has spoken: 

- I think this should probably be split up into multiple files. In general, I tend to think anything that depends on that many other things is probably doing too much itself. (regarding mostly this file and 'browser/js/components/Comment/index.js')

- Some of those might make sense as globals though, like maybe `Paper`